llvm/lib/Target/EVM/EVMControlFlowGraph.h

```diff
@@ -168,6 +168,12 @@ struct CFG {
 
   struct BuiltinCall {
     MachineInstr *Builtin = nullptr;
+    // This contains commutable operand indexes, if any.
+    // Operand indexes correspond to a stack layout, i.e., the first use
+    // operand in the instruction definition is located on the stack top
+    // and has zero index.
+    std::optional<std::pair<unsigned, unsigned>> CommutableOpIndexes =
```

Comment:
It is set to `{0, 1}` in `ControlFlowGraphBuilder::getCommutableOpIndexes` but it's never updated. Can't we use a flag instead?

Reply:
Sure we can, I just initially wanted to make it a bit more generalized.